components/TheFooter.vue

```diff
@@ -10,7 +10,7 @@
     is-retro
     is-upper-case
     label="Reset Game"
-    @click="resetGame('exit')"
```

Comment:
with the parameter set to 'exit' you will always land in the if statement  if(mode === "exit")...
the code in the statement will be executed